contracts/eosio.bios/include/eosio.bios/eosio.bios.hpp

```diff
@@ -165,6 +165,22 @@ namespace eosio {
          };
 
          typedef eosio::multi_index< "abihash"_n, abi_hash > abi_hash_table;
+         
+         using newaccount_action = action_wrapper<"newaccount"_n, &bios::newaccount>;
+         using updateauth_action = action_wrapper<"updateauth"_n, &bios::updateauth>;
+         using deleteauth_action = action_wrapper<"deleteauth"_n, &bios::deleteauth>;
+         using unlinkauth_action = action_wrapper<"unlinkauth"_n, &bios::unlinkauth>;
+         using canceldelay_action = action_wrapper<"canceldelay"_n, &bios::canceldelay>;
+         using onerror_action = action_wrapper<"onerror"_n, &bios::onerror>;
```

Comment:
No need for `onerror`. No one should be trying to send an inline `eosio::onerror` action.